views/includes/corporate-officer.html

```diff
@@ -1,6 +1,6 @@
 <div class="govuk-summary-card govuk-!-margin-bottom-4">
     <div class="govuk-summary-card_heading govuk-!-padding-top-4 govuk-!-padding-bottom-4 govuk-!-padding-right-3 govuk-!-padding-left-3" style="display: flex; justify-content: space-between; align-items: center">
-        <p class="govuk-!-font-weight-bold govuk-!-margin-0">{{officer.surname}}, {{officer.forename}}</p>
+        <p class="govuk-!-font-weight-bold govuk-!-margin-0">{{officer.surname}}</p>
```

Comment:
A bit odd (well, confusing) that the corporate name comes back / is placed in the 'surname' field but anyway, this change does look correct.